docs/index.asciidoc

```diff
@@ -209,7 +207,7 @@ mutate {
 
 [id="plugins-{type}s-{plugin}-json_key_file"]
 ===== `json_key_file`
-
+new[4.0.0, Replaces <<plugins-{type}s-{plugin}-key_password>>, <<plugins-{type}s-{plugin}-key_path>> and <<plugins-{type}s-{plugin}-service_account>>]]
```

Comment:
I think you might have an extra end brakcet here.

Reply:
Good catch!